deny.toml

```diff
@@ -86,6 +86,7 @@ allow = [
     "ISC",
     "MIT",
     "Unicode-DFS-2016",
+    "Unicode-3.0",
```

Comment:
nit: shouldn't "3" come before "D"?

Reply:
It can be in any order you like :). But yeah, probably a good idea to keep it sorted.